frontend/src/main/components/RecommendationRequest/RecommendationRequestForm.js

```diff
@@ -0,0 +1,229 @@
+                required: true,
+              })}
+            />
+            <Form.Control.Feedback type="invalid">
+              {errors.requesterName && "Requester Name is required. "}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Col>
+
+        {/* Professor Email */}
+        <Col>
+          <Form.Group className="mb-3">
+            <Form.Label htmlFor="professorEmail">Professor Email</Form.Label>
+            <Form.Control
+              data-testid="RecommendationRequestForm-professorEmail"
+              id="professorEmail"
+              type="email"
+              isInvalid={Boolean(errors.professorEmail)}
+              {...register("professorEmail", {
+                required: true,
+              })}
+            />
+            <Form.Control.Feedback type="invalid">
+              {errors.professorEmail && "Professor Email is required. "}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Col>
+
+        {/* Professor Name */}
+        <Col>
+          <Form.Group className="mb-3">
+            <Form.Label htmlFor="professorName">Professor Name</Form.Label>
+            <Form.Control
+              data-testid="RecommendationRequestForm-professorName"
+              id="professorName"
+              type="text"
+              isInvalid={Boolean(errors.professorName)}
+              {...register("professorName", {
+                required: true,
+              })}
+            />
+            <Form.Control.Feedback type="invalid">
+              {errors.professorName && "Professor Name is required. "}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Col>
+      </Row>
+
+      <Row>
+        {/* Request Type */}
+        <Col>
+          <Form.Group className="mb-3">
+            <Form.Label htmlFor="requestType">Request Type</Form.Label>
+            <Form.Select
+              data-testid="RecommendationRequestForm-requestType"
+              id="requestType"
+              type="string"
+              isInvalid={Boolean(errors.requestType)}
+              {...register("requestType", {
+                required: "Request Type is required.",
+              })}
+            >
+              <option value="">-- Select an Option --</option>
+              <option value="CS Department BS/MS program">
+                CS Department BS/MS program
+              </option>
+              <option value="Scholarship or Fellowship">
+                Scholarship or Fellowship
+              </option>
+              <option value="MS program (other than CS Dept BS/MS)">
+                MS program (other than CS Dept BS/MS)
+              </option>
+              <option value="PhD program">PhD program</option>
+              <option value="Other">Other</option>
+            </Form.Select>
+            <Form.Control.Feedback type="invalid">
+              {errors.requestType?.message}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Col>
```

Comment:
```suggestion
   {/* Replace with Dropdown selector that shows all values in the RequestType table */}
```

No.  The whole point of having a RequestType database table is that the request types are NOT hard coded.   This should be a dropdown of values from that table.